x-pack/filebeat/module/suricata/_meta/docs.asciidoc

```diff
@@ -0,0 +1,42 @@
+:modulename: suricata
+
+== suricata module
+
+This is the suricata module.
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+This module requires the
+{elasticsearch-plugins}/ingest-geoip.html[ingest-geoip] Elasticsearch plugin.
+
+This module has been developed against Suricata v4.0.4, but is expected to work with other versions of Suricata.
+
+
+include::../include/running-modules.asciidoc[]
+
+[float]
+=== Example dashboard
+
+This module comes with a sample dashboard. For example:
+
+TODO: include an image of a sample dashboard
```

Comment:
The docs still have some work to be done. But we can update them in a separate PR. From what I see they are not yet included in the Filebeat docs tree.

Reply:
@dedemorton This might be the first documentation for an X-Pack only module. Should I simply link it into the [filebeat/docs/module_list.asciidoc](https://github.com/elastic/beats/blob/ace94f0642a03aac0a94704bb8600b28be253418/filebeat/docs/modules_list.asciidoc).

Reply:
@andrewkroh You should use the x-pack role to flag X-pack features.

`[role="xpack"]`

Here's an example where we use the flag: https://github.com/elastic/beats/edit/6.4/libbeat/docs/security/securing-beats.asciidoc. 

AFAIK modules_list.asciidoc is a generated file, so your module should get added when you run `make update`.